metadata-mapper/metadata_mapper/mapper.py

```diff
@@ -76,3 +76,16 @@ def __init__(self, col_id, record):
     # Mapper Helpers
     def collate_subfield(self, field, subfield):
         return [f[subfield] for f in self.source_metadata.get(field, [])]
+
+    def collate_fields(self, fieldlist):
```

Comment:
YAY! Was this way of thinking about things useful? I'm really happy to see you using it.